src/lando/dockerflow/tests/test_dockerflow.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+from lando.tests.test_version import generate_version_file  # noqa: F401
```

Comment:
I think this should be removed in lieu of adding the fixtures to `conftest`. Fixture should probably go in the project-level conftest but can be imported in `lando.dockerflow.conftest`.

Reply:
Done.